lib/server.js

```diff
@@ -101,8 +104,17 @@ exports.create = function create(createOptions) {
         });
       }
       else {
+        // normalzie absolute view file path and absolute base directory path
+        // to prevent path strings like `/folder1/folder2/../../folder3/exampleFile`
+        thing = path.normalize(thing);
+        options.settings.views = path.normalize(options.settings.views);
```

Comment:
I didn't view the whole codes, but I think it's a really good idea.

so who will call the `render` function here: https://github.com/paypal/react-engine/blob/master/lib/server.js#L48

I need to change the parameters `thing` and `option` before they are passing into `render`